torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1070,12 +1066,15 @@ def __init__(
     def call_function(
         self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
     ) -> "VariableTracker":
-        from . import ConstantVariable, UserFunctionVariable
+        from . import UserFunctionVariable
         from .builder import wrap_fx_proxy
 
         tracer = self.fwd_bwd_tracer
 
-        self.check_kwargs(kwargs, ConstantVariable)
+        if len(kwargs) > 0:
+            unimplemented(
+                "kwargs have not been implemented for torch.autograd.Function"
+            )
```

Comment:
Tell me a bit more about this. Previously, we were checking if kwargs are constant. Now, if they exist, and are not constant, then we error out. This seems like a regression, unless we're saying that there is no way for kwargs to be constant before.

Reply:
Well the real issue is that even if we check kwargs are constant, we never pass it to speculate subgraph.

Reply:
Hence it would be inconsistent to record it as a proxy call kwargs when we never even record it in speculate subgraph.

Reply:
Lol that's fair